.github/workflows/truffle-ruby-test.yml

```diff
@@ -0,0 +1,155 @@
+name: build-with-truffleruby-head
+
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron: "30 14 * * *"
+
+jobs:
+  ruby-versions:
+    uses: ruby/actions/.github/workflows/ruby_versions.yml@master
+    with:
+      engine: truffleruby
```

Comment:
I may be wrong, but I don't think there's a truffleruby engine.

Reply:
Actually, I'm wrong on being wrong 🤦‍♂️ 
If we use the `setup-ruby` action, then `truffleruby` is indeed supported. But since we're using [`ruby_versions`](https://github.com/ruby/actions/blob/master/.github/workflows/ruby_versions.yml), I think only [these](https://cache.ruby-lang.org/pub/misc/ci_versions/) are supported.